pkg/image/utilities/utilities.go

```diff
@@ -730,17 +774,27 @@ if ($help)
 
 if (-not (Test-Path $image_list))
 {
-	echo "Could not find '$image_list' file"
+    echo "Could not find '$image_list' file"
     usage
-	exit 1
+    exit 1
+}
+
+if ([string]::IsNull($source_registry))
+{
+    $source_registry = ""
+}
+elseif (-not [string]::IsNullOrEmpty($source_registry))
+{
+    $source_registry += "/"
```

Comment:
this is not working in my setup
<img width="972" alt="Screen Shot 2022-11-09 at 10 30 36 AM" src="https://user-images.githubusercontent.com/16691014/200912239-622581e3-f382-4190-84ee-c81215fce297.png">

Reply:
Fixed!